### PR TITLE
g2psandbox Chart host/deploy

### PR DESCRIPTION
**Pull request by somanath21**
(https://mifosforge.jira.com/browse/GOV-412) Description

* Separate g2psandbox Chart deployable on its own 
* Dependency Chart is ph-ee-engine
* Add a link to the design document or include the design bullet points related to this PR here.

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: helm/g2p-sandbox/values.yaml [hunk 1]
```diff
@@ -146,7 +155,20 @@ ph-ee-engine:
...
     ams_local_auth_host: "https://fynams.sandbox.mifos.io/"
     ingress:
-      enabled: false    
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ams-mifos.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:164]: keep this as mifos and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 2]
```diff
@@ -156,10 +178,23 @@ ph-ee-engine:
     image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-mojaloop"
     imageTag: "latest"
     SPRING_PROFILES_ACTIVE: "bb"
-    hostname: "mojaloop.sandbox.mifos.io"
+    hostname: "mojaloop.sandbox.fynarfin.io"
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:181]: keep this as mifos and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 3]
```diff
@@ -193,15 +229,36 @@ ph-ee-engine:
...
       tenant: "gorilla"
     ingress:
-      enabled: false
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: channel.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:238]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 4]
```diff
@@ -193,15 +229,36 @@ ph-ee-engine:
...
+      hosts:
+        - host: channel.sandbox.fynarfin.io          
+          paths: 
+          - path: "/"
+            backend:
+              service:
+                name: "ph-ee-connector-channel"
+                port:
+                  number: 80
+        - host: channel-gsma.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:246]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 5]
```diff
@@ -211,26 +268,51 @@ ph-ee-engine:
...
       schema: "tenants"
     ingress:
-      enabled: false           
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops-bk.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:277]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 6]
```diff
@@ -254,25 +336,45 @@ ph-ee-engine:
...
-    zeebe_broker_contactpoint: "zeebe-zeebe-gateway:26500"
     ingress:
-      enabled: false            
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: notifications.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:359]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 7]
```diff
@@ -254,25 +336,45 @@ ph-ee-engine:
...
 
   connector_bulk:
     enabled: true
     image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-bulk-processor"
-    imageTag: latest
+    imageTag: "latest"
+    zeebe_broker_contactpoint: "g2pconnect-zeebe-gateway:26500"
     tenant: "rhino,gorilla"
-    hostname: bulk-connector.sandbox.mifos.io
+    hostname: bulk-connector.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:377]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 8]
```diff
@@ -282,41 +384,81 @@ ph-ee-engine:
...
         batch_transaction: "/api/v1/batch/transactions"
     ingress:
-      enabled: false
+      enabled: true
+      annotations: 
+        konghq.com/plugins: request-transformer,oidc,cors
+      tls:
+      - secretName: sandbox-secret
+        hosts:
+          - bulk-connector.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:393]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 9]
```diff
@@ -282,41 +384,81 @@ ph-ee-engine:
...
     tenants: "rhino,gorilla"
     ingress:
-      enabled: false            
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: zeebeops.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:418]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 10]
```diff
@@ -325,11 +467,20 @@ ph-ee-engine:
...
+    imageTag: "latest"
     LOGGING_LEVEL_ROOT: "DEBUG"
 
   keycloak:
     enabled: true
+    ingress:
+      enabled: true
+      ingressClassName: "kong"
+      rules:
+        - host: 'keycloak.sandbox.fynarfin.io'
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:479]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 11]
```diff
@@ -346,4 +497,4 @@ ph-ee-engine:
         enabled: false
       ingress:
         enabled: true
-        hostname: admin-kong.sandbox.mifos.io
+        hostname: admin-kong.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:500]: keep this as mifos.io and change from cookbooks

File: helm/g2p-sandbox/values.yaml [hunk 12]
```diff
@@ -211,26 +268,51 @@ ph-ee-engine:
...
     hostname: "ops.sandbox.mifos.io"
+    SPRING_PROFILES_ACTIVE: "bb"
     ingress:
-      enabled: false         
+      enabled: true 
+      className: "kong"
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: ops.sandbox.fynarfin.io
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:301]: keep this as mifos.io and change from cookbooks

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/g2p-sandbox/requirement.yaml [hunk 13]
```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: ph-ee-engine
+  repository: https://fynarfin.io/images/
+  version: 0.0.0-SNAPSHOT
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/g2p-sandbox/requirement.yaml:4]: pointed to ph-ee-engine-0.0.0-SNAPSHOT to work circleCI test.

File: helm/g2p-sandbox/values.yaml (hunk 1 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

File: helm/g2p-sandbox/values.yaml (hunk 2 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: changed

File: helm/g2p-sandbox/values.yaml [hunk 14]
```diff
@@ -282,41 +384,81 @@ ph-ee-engine:
...
     hostname: "messagegateway.sandbox.mifos.io"
     ingress:
-      enabled: false   
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx  
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: messagegateway.sandbox.fynarfin.io
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: changed

File: helm/g2p-sandbox/values.yaml (hunk 11 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

File: helm/g2p-sandbox/values.yaml [hunk 15]
```diff
@@ -146,7 +155,20 @@ ph-ee-engine:
     ams_local_customer_host: "https://fynams.sandbox.mifos.io/"
     ams_local_auth_host: "https://fynams.sandbox.mifos.io/"
     ingress:
-      enabled: false    
+      enabled: true
```
Comment (avikganguly01) [inline helm/g2p-sandbox/values.yaml:158]: Parent charts shouldn't have ingresses enabled as true.

File: helm/ph-ee-engine/connector-ams-mifos/values.yaml [hunk 16]
```diff
@@ -51,4 +51,4 @@ operations_app:
   datasource:
     host: "operationsmysql"
 keycloak:
-  enabled: true 
\ No newline at end of file
+  enabled: "true"
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/ph-ee-engine/connector-ams-mifos/values.yaml]: - Removed realm-secret from ams-mifos. 
- Keycloak enabled: true not required.

File: helm/g2p-sandbox/values.yaml (hunk 12 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

File: helm/g2p-sandbox/values.yaml (hunk 10 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

File: helm/g2p-sandbox/values.yaml [hunk 17]
```diff
@@ -156,10 +178,23 @@ ph-ee-engine:
...
     DFSPIDS: "gorilla,lion"
     ingress:
-      enabled: false           
+      enabled: true
+      annotations: 
+        kubernetes.io/ingress.class: nginx      
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: mojaloop.sandbox.fynarfin.io
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

File: helm/g2p-sandbox/values.yaml [hunk 18]
```diff
@@ -44,13 +58,12 @@ ph-ee-engine:
       annotations: 
         kubernetes.io/ingress.class: nginx      
       path: /
-      host: operate.sandbox.mifos.io
+      host: operate.sandbox.fynarfin.io
```
[the remark this answers is not in the document]
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Changed

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: helm/g2p-sandbox/values.yaml (hunk 15 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Disabled Ingresses for all the services

**Pull request merged**